caf-logging-log4j2/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
ElasticSearch doesn't work with an xml file. So add added a properties version too.

Reply:
Why is that?  Is it not just that Elasticsearch ships with a properties file that needs to be deleted so that it falls back to looking for the xml file?